__tests__/services/TransactionFilterService.spec.ts

```diff
@@ -17,6 +17,8 @@ import { TransactionFilterOptionsState, TransactionState } from '@/store/Transac
 import { TransactionFilterService } from '@/services/TransactionFilterService';
 import { getTestAccount } from '@MOCKS/Accounts';
 import { TransferTransaction } from 'symbol-sdk';
+import { addressBookMock } from '@MOCKS/AddressBookMock';
+import { IContact } from 'symbol-address-book';
 
 const currentSigner = getTestAccount('remoteTestnet');
 const recepient = getTestAccount('remoteTestnetRecipient');
```

Comment:
1. not something you need to fix as part of this PR, but it's a bit confusing to use currentSigner as recipient address
2. recepient is spelled wrong